### PR TITLE
AmpTask1310_Implement_RH1E

### PR DESCRIPTION
**Pull request by gpsaggese**

### CODE REVIEW

File: core/dataflow/nodes/transformers.py [hunk 1]
```diff
@@ -367,7 +367,8 @@ def __init__(
         nan_mode: Optional[str] = None,
     ) -> None:
         """
-        For reference, let
+        For reference, let:
```
Comment (gitpaulsmith) [inline core/dataflow/nodes/transformers.py:370]: These incorrect linter changes keep flicking the code back and forth. I would stop the linter from making these changes in the first place.

File: core/dataflow/nodes/transformers.py [hunk 2]
```diff
@@ -510,6 +512,7 @@ def _transform(
         self, df: pd.DataFrame
     ) -> Tuple[pd.DataFrame, collections.OrderedDict]:
         # Preprocess to extract relevant flat dataframe.
+        # TODO(gp): -> df_orig otherwise it gets confused with df_in (vs df_out).
```
Comment (gitpaulsmith) [inline core/dataflow/nodes/transformers.py:515]: What is `df_orig`?

**Review by gitpaulsmith — COMMENTED** [review]
I have a lot of questions about the changes here. Is this just a stated WIP draft, or is this going to become a candidate for merging?

**Comment by gpsaggese** [comment]
> I have a lot of questions about the changes here. Is this just a stated WIP draft, or is this going to become a candidate for merging?

It is a draft (in fact it is grayed out), so no need to review.

I am going to go through it and see what's interesting and do a real PR asking for a review, if there is something interesting inside. If it's something potentially controversial we'll talk about it before a PR. It's the usual process.

**Comment by gpsaggese** [comment]
Nothing interesting left here.

**Pull request closed without merging**